guides/common/modules/con_backing-up-server-and-proxy.adoc

```diff
@@ -22,7 +22,7 @@ You must encrypt or move the backup to a secure location to minimize the risk of
 .Conventional Backup Methods
 You can also use conventional backup methods.
 ifndef::orcharhino[]
-For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/configuring_basic_system_settings/index#assembly_recovering-and-restoring-a-system_configuring-basic-system-settings[Recovering and restoring a system] in the _{RHEL}{nbsp}8 Configuring Basic System Settings Guide_.
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/configuring_basic_system_settings/index#assembly_recovering-and-restoring-a-system_configuring-basic-system-settings[Recovering and restoring a system] in _{RHEL}{nbsp}8 Configuring Basic System Settings_.
```

Comment:
**Configuring basic system settings** Is the official name of the guide, so I don't know whether to include RHEL 8. Also, unlike us, this guide does not use camel case headings. Should this be reflected in link as well? Thoughts?

Reply:
Capitalization - agreed. I would lean towards keeping RHEL 8 to signify that it's about a different product than Satellite.